en/docs/Learn/APIMonetization/monetizing-an-api.md

```diff
@@ -0,0 +1,703 @@
+# Monetizing an API
+
+API Monetization is the process that allows API publishers to earn money by charging developers for using their services that have been exposed as APIs. When using API Monetization, API publishers can select billing plans based on their requirements and API providers are able to maximize their revenue by selecting the most profitable pricing plan for their APIs.
+
+WSO2 provides a API Monetization solution via WSO2 API Manager (WSO2 API-M) that allows you to generate revenue off your APIs based on your business
```

Comment:
Shall we change this section to say that WSO2 API Manager offers this solution? Rather than saying WSO2 offers via WSO2 API Manager? Or maybe we can add it to the first paragraph.

Reply:
The introduction has been redone.